_config.yml

```diff
@@ -28,13 +28,13 @@ github_username: ubccr
 github_url: https://github.com/ubccr/xdmod-appkernels
 xdmod_module_name: Application Kernels
 latest_version: "10.5"
+latest_sw_version: "10.5.0"
 version_list:
   - "10.5"
-  - "10.0"
 unsupported_version_list:
+  - "10.0"
   - "9.5"
   - "9.0"
-  - "8.5"
```

Comment:
Recommend leaving this in since removing will 'break' the 8.5 pages due to the way our xdmod-jekyll-theme works.

Reply:
I'm not sure how it will break it. For example, `8.0` is not currently in the list, but [the 8.0 page](https://appkernels.xdmod.org/8.0/ak-overview.html) looks similar to [the 8.5 page](https://appkernels.xdmod.org/8.5/ak-overview.html).

Reply:
It looks like the versions for [9.0](https://appkernels.xdmod.org/9.0/ak-overview.html) and prior are missing the menu because those versions are not listed in the `defaults` in `_config.yml`; I have updated this.